packages/tabsheet/src/vaadin-tabsheet.d.ts

```diff
@@ -3,8 +3,94 @@
  * Copyright (c) 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
+import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
+import type { Tab } from '@vaadin/tabs/src/vaadin-tab';
```

Comment:
nit: let's use `.js` extension as for other imports:

```suggestion
import type { Tab } from '@vaadin/tabs/src/vaadin-tab.js';
```